content/docs/features/data-fetching.md

```diff
@@ -4,22 +4,20 @@ id: '/docs/features/data-fetching'
 next: '/docs/tinacms-context'
 ---
 
-There are essentially two touch points you'll need set up in order for Tina to work with your Next.js application or website.
+With Tina, your content is stored in Git along with your codebase. Instead of using something like `fs.readfile` to access your content, Tina provides an API to query your content through GraphQL, based on your defined content models.
```

Comment:
I'm not sure if this needs to be said:

> Instead of using something like `fs.readfile` to access your content,

Folks coming from most headless content management systems will be more comfortable with API fetching vs fetching from a file. For folks who are accustomed to fetching from a file, I think the difference could be explained later so that they aren't caught up in the differences so early into the doc.

Reply:
```suggestion
With Tina, your content is stored in Git along with your codebase. Tina provides an API to query your content through GraphQL, based on your defined content models.
```